templates/global/filter_form.html

```diff
@@ -1,51 +1,50 @@
 {% block filter_form %} {% load static %}
-    <link rel="stylesheet" type="text/css" href="{% static 'css/form.css' %}"/>
-    <h2 class="form_title">House Filter</h2>
+<h3 class="text-center text-black pt-1">House Filter</h3>
 
-    <form
-            method="post"
-            id="HouseFilter"
-            data-cities-url="{% url 'house:ajax_load_cities' %}"
-            novalidate
-    >
-        <table>
-            <tr>
-                {{ form.as_table }}
-            </tr>
-            <tr>
-                <td>
-                    <button type="submit">Submit</button>
-                </td>
-            </tr>
-        </table>
-        {% csrf_token %}
-    </form>
+<form
+  method="post"
+  id="HouseFilter"
+  data-cities-url="{% url 'house:ajax_load_cities' %}"
+  novalidate
+>
+  <table>
+    <tr>
+      {{ form.as_table }}
+    </tr>
+    <tr>
+      <td>
+        <button type="submit" class="btn btn-info btn-md">Submit</button>
+      </td>
+    </tr>
+  </table>
+  {% csrf_token %}
+</form>
 
-    <script>
-        const url = $("#HouseFilter").attr("data-cities-url");
+<script>
+  const url = $("#HouseFilter").attr("data-cities-url");
 
-        $(document).ready(function () {
-            change_cities($("#id_country").val());
-        });
+  $(document).ready(function () {
+    change_cities($("#id_country").val());
+  });
 
-        $("#id_country").change(function () {
-            change_cities($(this).val());
-        });
+  $("#id_country").change(function () {
+    change_cities($(this).val());
+  });
 
-        function change_cities(countryId) {
-            if (countryId === "") return;
-            $.ajax({
-                url: url,
-                data: {
-                    country: countryId,
-                },
-                success: function (data) {
-                    var id = $("#id_city").val();
-                    $("#id_city").html(data);
-                    document.querySelector("#id_city").value = id;
-                },
-            });
-        }
-    </script>
+  function change_cities(countryId) {
+    if (countryId === "") return;
+    $.ajax({
+      url: url,
+      data: {
+        country: countryId,
+      },
+      success: function (data) {
+        var id = $("#id_city").val();
+        $("#id_city").html(data);
+        document.querySelector("#id_city").value = id;
+      },
+    });
+  }
+</script>
```

Comment:
I know that it was already merged and you just fixed the indentation, but it is a bad practice having logic inside the html file since it can't be tested properly.
I would recommend on refactoring it in the future and getting all the code that's inside html files into dedicated python files with the relevant tests.
@shlior7 @ronyyosef

Reply:
This is frontend logic
if this was react or any other frontend framework this logic would be there I can split the js and the html to different files but I don't see how I can implement this frontend logic(changing the city select options when the country select option changes) could be done from the backend code i.e. python file and have tests for it.
If there is a way to do that I would love to hear it

Reply:
I found a library ([smart-select](https://github.com/jazzband/django-smart-selects)) that could do this automagiclly I played with it and it seems complicated I'll try to implement this in a different PR and let you know

Reply:
OK, lets see how it goes.